docs/source/user/tutorial.rst

```diff
@@ -33,7 +33,7 @@ Start FSCrawler
    # On Linux/Mac
    bin/fscrawler resumes
    # On Windows
-   .\bin\fscrawler resumes
+   .\bin\fscrawler\ resumes
```

Comment:
I don't think you should change this... That's the "binary" name `fscrawler.bat` and not a directory.